### PR TITLE
[ci] Add styling to PR status table

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

File: ci2/ci/templates/index.html [hunk 1]
```diff
@@ -2,6 +2,24 @@
   <head>
     <title>CI</title>
     <link rel="stylesheet" type="text/css" href="/static/style.css">
+    <style>
+      #statuses {
+      border-collapse: collapse;
```
Comment (akotlar) [inline ci2/ci/templates/index.html]: Indentation

**Comment by tpoterba** [comment]
fixed

**Comment by tpoterba** [comment]
we don't have a way to test this right now, right? (we could write something that templates with mock data)

I want to next add something that colors the status link - but this will involve a bit of jinja language which I'll probably get wrong at first.

**Comment by akotlar** [comment]
> we don't have a way to test this right now, right? (we could write something that templates with mock data)
> 
> I want to next add something that colors the status link - but this will involve a bit of jinja language which I'll probably get wrong at first.

Yeah, we have no ui-related tests. Jackie had previously made a suggestion to output the rendered web page as a build artifact, which would be nice.

**Comment by akotlar** [comment]
JS fiddle is pretty nice: https://jsfiddle.net/a4dw5kf2/1/

**Pull request merged**